blogposts/2021/release-3.28.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Sourcegraph 3.28 release"
+publishDate: 2021-05-20T10:00-07:00
+description: "Sourcegraph 3.28 release"
+tags: [blog, release]
+slug: "release/3.28"
+published: true
+changelogItems:
+  # Added
+  - description: "Added `select:commit.diff.added` and `select:commit.diff.removed` for `type:diff` search queries. These selectors return commit diffs only if a pattern matches in `added` (respectively, `removed`) lines."
+    url: https://github.com/sourcegraph/sourcegraph/pull/20328
+    category: Search
+  - description: "Additional language autocompletions for the `lang:` filter in the search bar."
+    url: https://github.com/sourcegraph/sourcegraph/pull/20535
+    category: Search
+  - description: "Steps in batch specs can now have an `if:` attribute to enable conditional execution of different steps."
+    url: https://github.com/sourcegraph/sourcegraph/pull/20701
+    category: Batch Changes
+  - description: "Extensions can now log messages through `sourcegraph.app.log` to aid debugging user issues."
+    url: https://github.com/sourcegraph/sourcegraph/pull/20474
+    category:
+  # Changed
+  - description: "User and site credentials used in Batch Changes are now encrypted in the database if encryption is enabled with the `encryption.keys` config."
```

Comment:
@christinaforney - is this worth keeping?

Reply:
I don't think it's worth including this at this level.